app/presenters/users_presenter.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+require "#{Rails.root}/lib/word_count"
+
+#= Presenter for users view
+class UsersPresenter
+  def initialize(user:)
+    @user = user
+  end
+
+  def individual_courses
+    @user.courses.where('courses_users.role = 0')
```

Comment:
Instead of the "magic number" 0 here, you should use the constant, `CoursesUsers::Roles::STUDENT_ROLE`.